app/Http/Controllers/SitemapController.php

```diff
@@ -66,11 +66,11 @@ public function board(){
 
     }
 
-    public function news(){
+    public function news() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->